File: Python/Project_Euler_S9.py
# Project Euler – Problem 9
# Title: Special Pythagorean Triplet
# Link: https://projecteuler.net/problem=9
#
# Find the Pythagorean triplet for which
# a + b + c equals a given value.


for a in range(1, 1000):
    for b in range(a + 1, 1000):
        c = 1000 - a - b
        if a**2 + b**2 == c**2:
            print("a:", a, "b:", b, "c:", c)
            print("Product of abc:", a * b * c)

File: C/Project_Euler_S9.c
// Project Euler – Problem 9
// Title: Special Pythagorean Triplet
// Link: https://projecteuler.net/problem=9
//
// Find the Pythagorean triplet for which
// a + b + c equals a given value.

#include <stdio.h>

int main()
{
    for (int a = 1; a < 1000; a++)
    {
        for (int b = a + 1; b < 1000; b++)
        {
            int c = 1000 - a - b;
            if ((a*a) + (b*b) == c*c)
            {
                printf("a: %d b: %d c: %d \n", a, b, c);
                printf("Product of abc: %d", a*b*c);
            }
        }
    }
    return 0;
}

